dotcom-rendering/src/web/components/SecureSignup.tsx

```diff
@@ -92,7 +61,11 @@ const generateForm = (
 	return { html, styles };
 };
 
-export const SecureSignup = ({ newsletterId, successDescription }: Props) => {
+export const SecureSignup: React.FC<Props> = ({
```

Comment:
We don't use the ` React.FC<Props>` style of typing components in DCR. There's a [nice write up for why in this article](https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript) but essentially it's mainly because it adds a `children` prop which then isn't used but also because we want to keep one way for how things are done.